test/dummy/config/application.rb

```diff
@@ -8,7 +8,7 @@
 module Dummy
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.0
+    config.load_defaults ENV["APPRAISAL"] == "5.2" ? 5.2 : 6.0
```

Comment:
I think we can just delete this in order to always test against the defaults for the currently running Rails version. That should make it simpler. 👍

Reply:
This cleans up a couple of lines, and no more goofing around with the environment variable.